boards/opentitan/earlgrey-cw310/README.md

```diff
@@ -0,0 +1,7 @@
+OpenTitan ChipWhisper CW310 Board
```

Comment:
This reads as if OpenTitan is the producer of the CW310, which is not correct. Something like "OpenTitan on the ChipWhisperer CW310 Board" would be a little clearer.

Reply:
Done.